.env_sample

```diff
@@ -1,3 +1,3 @@
 TOKEN=[ENTER_YOUR_TOKEN_HERE]
-DEFAULT_GUILD=953312435337580584
-LOG_LEVEL=debug
\ No newline at end of file
+DEFAULT_GUILD=[ENTER_YOUR_DEFAULT_GUILD_HERE]
```

Comment:
i'd prefer the old one, as all bots are most likely gonna test on the dedicated test server. 
that way, one already has stuff set up, and we can just change it for the production bot

Reply:
I mean this is the sample one

Reply:
yeah is there something special about `953312435337580584` ?

Reply:
Yeah its the id of the test bot guild we all have our bots on

Reply:
then we just dont need it in the .sample_env
the .sample_env is for people on the outside to understand how to fill the .env

Reply:
Well having an example helps to understand how it works too. 

Also most people using this will be us.